src/Aspire.Dashboard/Model/DashboardClient.cs

```diff
@@ -34,8 +37,6 @@ namespace Aspire.Dashboard.Model;
 /// </remarks>
 internal sealed class DashboardClient : IDashboardClient
 {
-    private const string ResourceServiceUrlVariableName = "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL";
```

Comment:
Setting the resource service won't be very common. I replaced it with `Dashboard:ResourceServiceClient:Url`.

Reply:
Why won't it be common?

Reply:
Let me rephrase. It won't be common for people to start the dashboard with their own resource service. Aspire tooling will (DCP) and some cloud hosters might setup a resource service, but people just playing around with the dashboard won't.

I removed it to keep envvar/cmd arg optimized variables like this focused on the most commonly used settings (endpoint addresses and endpoint security). Do you want it added back?

Reply:
Yea I think we should add it back. It's the other primary function of the dashboard. We'll have other implementations soon.